internal/p2p/uploads.go

```diff
@@ -40,24 +40,24 @@ func ProcessUploads(folderPath string) {
 						fmt.Println("[INFO] File Security checking file: " + f + " with SHA256: " + fsha256)
 						var result bool
 						var err error
-						// TODO: 				if appconfInstance.SecurityCapability [1 2 3 4]
-						if appconfInstance.SecurityCapability <= 3 {
+
+						// Skip all security scanning if FFSkipVT is enabled
+						if appconfInstance.FFSkipVT {
+							fmt.Println("[INFO] Virus scanning disabled (FFSkipVT=true), skipping security check")
+							result = true
+						} else if appconfInstance.SecurityCapability <= 3 {
```

Comment:
Again, it seems like a switch statement (or a set thereof) would make this cleaner. It also seems like we're using magic numbers (not sure if they're the same) - let's switch to enum consts plesae.